src/services/ldap-config/docs/openapi.yaml

```diff
@@ -28,6 +28,12 @@ components:
       in: query
       schema:
         type: boolean
+    verifyFullSync:
+      name: verifyFullSync
+      description: load all users from LDAP server for verification (default = false)
+      in: query
+      schema:
+        type: boolean
```

Comment:
needs to be referenced in paths.post.parameters and paths.patch.parameters on lines 60 and 97

Reply:
done